steel_industry/functions.py

```diff
@@ -5,11 +5,14 @@
 
 
 def split_name(name):
-    '''
-    :param:  name: (string) - Entry in the “name” column of result.csv
-    :return: (list) of strings
-    :logik: divide a "_"-seperated string into a list of strings
-    '''
+    """
+    divide a "_"-seperated string into a list of strings, depending on last charakter is digit.
```

Comment:
Könntest du das genauer beschreiben? ich verstehe den Satz nicht ganz.

Reply:
da möchte ich sagen, dass hier ein String der mit "_" geteilt ist, in eine liste von string umgewandelt wird, dabei wird unterschieden ob das letzte Zeichen eine Zahl ist.